modules/client-configuration/pages/supported-features-sles.adoc

```diff
@@ -26,222 +26,266 @@ The icons in this table indicate:
 | Feature
 | {sle}{nbsp}12
 | {sle}{nbsp}15
-| {opensuse}{nbsp}15
+| {opensuse} Leap {nbsp}15
+| {opensuse} Leap {nbsp}16
 
 | Client
 | {check}
 | {check}
 | {check}
+| {check}
```

Comment:
For now I assumed openSUSE Leap 16.0 will get the same features, but this is to be confirmed, so do not merge this PR without checking!